cfpb/cfgov-refresh
# Updates requirements with security impacts

Updates requirement pinned versions of `PyYAML` and `requests` to fix security vulnerabilities. This affects `college-costs` as well as `cfgov-refresh`.

## Changes

- Updates PyYAML to 4.2b1 for security fixes
- Updates requests to 2.20.0 for security fixes

## Testing

1. Go to DEV5 (build5 server, only internally available)
1. Visit the College Costs tool `/paying-for-college/compare-financial-aid-and-college-cost/` and verify the tool still works.
1. Unit-tests
1. Verify site functionality works

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
